Sources/LiveViewNative/NavStackEntryView.swift

```diff
@@ -43,23 +43,19 @@ struct NavStackEntryView<R: RootRegistry>: View {
                         .id(ObjectIdentifier(document))
                 } else {
                     SwiftUI.Group {
-                        if R.LoadingView.self == Never.self {
-                            switch coordinator.state {
-                            case .connected:
-                                fatalError()
-                            case .notConnected:
-                                SwiftUI.Text("Not Connected")
-                            case .connecting:
-                                SwiftUI.ProgressView("Connecting")
-                            case .connectionFailed(let error):
-                                SwiftUI.VStack {
-                                    SwiftUI.Text("Connection Failed")
-                                        .font(.subheadline)
-                                    SwiftUI.Text(error.localizedDescription)
-                                }
+                        switch coordinator.state {
+                        case .connected, .reconnecting:
+                            fatalError()
+                        case .notConnected:
+                            SwiftUI.Text("Not Connected")
+                        case .connecting:
+                            SwiftUI.ProgressView("Connecting")
+                        case .connectionFailed(let error):
+                            SwiftUI.VStack {
+                                SwiftUI.Text("Connection Failed")
+                                    .font(.subheadline)
```

Comment:
I think we should remove this from being default, and that all default states should be `nil`. Instead we should move these into the template project generator so people can have direct access to modify.

Reply:
So should the state view arguments be required? Or do states like `connecting`, `disconnected`, etc. just render a blank screen if no argument is passed?

Reply:
No, not required. If they aren't provided it's just a `no-op`

Reply:
How about the LiveView error webpage View. Should we display that by default?

Reply:
oh that's a good point, how about the view is shipped with the client but we still have the code to render that view in the error closure generated into the template